src/rooms.ts

```diff
@@ -77,6 +79,19 @@ const roomsRequest = async ({
     console.log(`<<< ${method} ${url} ${status}`);
 
     if (!successCodes.includes(status)) {
+      if (createP && status === 409) {
+        console.log(`!!! retry ${method} ${url} ${status}`);
```

Comment:
Do you not need to put a delay in here?  e.g. use a `setTimeout` to wait say half a second before retrying.  Just worried this might effectively "hang" by getting into a tight retry loop.

Reply:
i don't think it can loop because the retry has `createP=false` ... hence, if it fails a second time, it won't retry for a third...

Reply:
Didn't spot that! Perfect :)